server/package.json

```diff
@@ -4,17 +4,20 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "start": "node index.js",
+    "start": "nodemon index.js",
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "bcrypt": "^5.1.0",
     "cors": "^2.8.5",
     "dotenv": "^16.0.3",
     "express": "^4.18.2",
-    "mongoose": "^6.7.3",
-    "node-fetch": "^3.3.0"
+    "jsonwebtoken": "^8.5.1",
+    "mongoose": "^6.8.0",
+    "node-fetch": "^3.3.0",
+    "nodemon": "^2.0.20"
```

Comment:
since it's not decided to add nodemon as a dependency, i recommend you install it globally if you want to use it